tests/readmes/test_erc20_enterprise.py

```diff
@@ -0,0 +1,266 @@
+#
+# Copyright 2021 Ocean Protocol Foundation
+# SPDX-License-Identifier: Apache-2.0
+#
+import json
+import os
+
+from ocean_lib.example_config import ExampleConfig
+from ocean_lib.models.models_structures import (
+    CreateErc20Data,
+    DispenserData,
+    OrderParams,
+)
+from ocean_lib.ocean.mint_fake_ocean import mint_fake_OCEAN
+from ocean_lib.ocean.ocean import Ocean
+from ocean_lib.web3_internal.constants import ZERO_ADDRESS
+from ocean_lib.web3_internal.wallet import Wallet
+
+
+def test_erc20_enterprise_flow_with_dispenser():
+    config = ExampleConfig.get_config()
+    ocean = Ocean(config)
+
+    # Create Alice's wallet
+    alice_private_key = os.getenv("TEST_PRIVATE_KEY1")
+    alice_wallet = Wallet(
+        ocean.web3,
+        alice_private_key,
+        config.block_confirmations,
+        config.transaction_timeout,
+    )
+    assert alice_wallet.address
+
+    # Mint OCEAN
+    mint_fake_OCEAN(config)
+    assert alice_wallet.web3.eth.get_balance(alice_wallet.address) > 0, "need ETH"
+
+    # Publish an NFT token
+    nft_token = ocean.create_nft_token("NFTToken1", "NFT1", alice_wallet)
+    token_address = nft_token.address
+    assert token_address
+
+    # Prepare data for ERC20 Enterprise token
+    cap = ocean.to_wei(200)
+    erc20_data = CreateErc20Data(
+        template_index=2,  # default value
```

Comment:
please adjust the comment here: it is not the default value now, it is the template corresponding to ERC20Enterprise